tools/editliveos

```diff
@@ -2346,7 +2482,7 @@ def main():
             editor._configure_bootloader(editor._LiveImageCreatorBase__isodir)
         if editor.ks:
             # Run_pre_scripts same code as ImageCreator_run_post_scripts
-            editor._run_post_scripts()
+            # editor._run_post_scripts()
```

Comment:
Why is this commented out?

Reply:
It was commented out because it runs the post-script twice, I saw a function called ```get_pre_scripts``` but there was no further usage of that function, I am not sure how to add a function that runs the pre-script. I can imitate the ```_run_post_scripts``` function but replaces ```get_pre_scripts``` with ```get_post_scripts```, but I'm not sure if that will work since there are other functions that are post-script specific.